CHANGELOG.md

```diff
@@ -7,6 +7,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Add
+
+- initial support for Cluster API Provider VSphere CRs:
+  - `capi_vspherecluster_annotation_paused`
```

Comment:
I'm wondering if those should not be named `capi_vsphere_(cluster|machine|vm)_created` instead of capi_vsphere(cluster|machine|vm)_created. @bavarianbidi what do you think?

Reply:
yes, i would like to follow the naming approach which is already there on upstream, where the `cluster` Info metric got named `capi_cluster_info` (xref [upstream config](https://github.com/kubernetes-sigs/cluster-api/blob/main/hack/observability/kube-state-metrics/crd-config.yaml#L19-L21))